database/migrations/2024_06_23_000000_create_failed_one_time_operations_table.php

```diff
@@ -19,6 +19,7 @@ public function up(): void
         Schema::create($this->name, function (Blueprint $table) {
             $table->id();
             $table->string('name');
+            $table->string('type');
```

Comment:
Failed Operation migration is already migrated. so we need to add extra migration to alter the table and add the type column and make sure in your alter migration the type column is not exists